Author: Pino Toscano <pino@debian.org>
Description: override GENERIC_LIB_VERSION & KDE_NON_GENERIC_LIB_VERSION
 Override the two cmake variables GENERIC_LIB_VERSION and
 KDE_NON_GENERIC_LIB_VERSION (set by KDE4Defaults.cmake, provided by kdelibs),
 to avoid adjusting .install files every time kdelibs is upgraded to a newer
 stable serie (e.g. 4.6.x -> 4.7.x).
Forwarded: not-needed
Last-Update: 2012-04-17

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -40,6 +40,10 @@
 include(MacroBoolTo01)
 find_package(KdepimLibs 4.4.1 REQUIRED)
 
+# temporarly override them to not depend on kdelibs
+set(GENERIC_LIB_VERSION "4.6.0")
+set(KDE_NON_GENERIC_LIB_VERSION "5.6.0")
+
 
 ############### search Boost ###############
 
